pettingzoo/mpe/_mpe_utils/simple_env.py

```diff
@@ -12,6 +12,7 @@
 from pettingzoo.utils.agent_selector import AgentSelector
 
 alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+DYNAMIC_RESCALING = True
```

Comment:
Apologies for necro-ing this PR. The devs for PettingZoo recently went on a short hiatus.

Could this not be a global variable and be an environment argument instead? Also default it to True so we don't break existing learned algorithms which utilize image space.